budgetportal/webflow/urls.py

```diff
@@ -21,5 +22,15 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^infrastructure-projects/provincial/(?P<id>\d+)-(?P<slug>[\w-]+)/csv-download$",
+        views.ProvInfaProjectCSVDownload.as_view(),
+        name="provincial-infra-project-detail-csv-download",
+    ),
     url(r"^api/v1/", include(router.urls)),
+    path(
+        "infrastructure-projects/provincial/search/get-csv",
+        views.ProvInfraProjectSearchView.as_view({"get": "get_csv"}),
```

Comment:
here is the view set thing. We still use the same class that is cool, we do not need to worry about having the same filters etc